packages/client/subscriptions/NotificationSubscription.ts

```diff
@@ -66,7 +68,9 @@ const subscription = graphql`
       ...ArchiveTimelineEventMutation_notification @relay(mask: false)
       ...SetNotificationStatusMutation_notification @relay(mask: false)
       ...CreateTaskMutation_notification @relay(mask: false)
+      ...EndCheckInMutation_notification @relay(mask: false)
       ...EndNewMeetingMutation_notification @relay(mask: false)
```

Comment:
can we remove this now, or should we wait a month or 2 before we remove?

Reply:
As we're keeping the old mutations, I was thinking that we may as well keep the references to the old mutations too. Then we can remove everything once the new mutations are up and running. I'd be happy to remove these references though if you think it'd be cleaner

Reply:
i like it! can you create a new issue like "remove old checkin mutation" that we can process in a month or 2?

Reply:
Sure thing 👍